src/app/components/client/UserMenu.tsx

```diff
@@ -5,21 +5,80 @@
 "use client";
 
 import { Session } from "next-auth";
-import { signIn } from "next-auth/react";
+import Image from "next/image";
+import { signOut } from "next-auth/react";
+import Script from "next/script";
+
 import { useL10n } from "../../hooks/l10n";
+import OpenInIcon from "../../../client/images/icon-open-in.svg";
+import SettingsIcon from "../../../client/images/icon-settings.svg";
+import HelpIcon from "../../../client/images/icon-help.svg";
+import SignOutIcon from "../../../client/images/icon-signout.svg";
 
 export type Props = {
   session: Session | null;
+  fxaSettingsUrl: string;
 };
 
-export const UserMenu = (props: Props) => {
+export const UserMenu = ({ session, fxaSettingsUrl }: Props) => {
```

Comment:
We’ll have to decide how we handle duplicate components best: The one we need/would like to have for the current site and then new ones that will be built in a “React way” for Premium.

Reply:
I'd suggest (like in a different comment in this PR) to keep everything that's directly copied over from the "old" site inside the `(nextjs_migration)` directory, and create new versions of these components outside of them. I'd like to avoid applying the global stylesheet from the old website inside the new one, and would rather incrementally recreate them using CSS modules.

So all the components in this PR that are inside `/components/client` would be moved, since they won't look right in the parts of the website we do "the React way".